public/graph.js

```diff
@@ -85,6 +86,7 @@ function retrieve_graph() {
 
   var deps_url = new URL('/api/v1/deps', window.location.href);
   deps_url.searchParams.set('dist', dist);
+  deps_url.searchParams.set('dist_version', dist_version);
```

Comment:
Would be better to only set this if dist_version was not null or empty string

Reply:
Done